<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotDialogUI</class>
 <widget class="QDialog" name="PlotDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="labelX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xLeftBox">
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>-10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xRightBox">
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yLeftBox">
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>-10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yRightBox">
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="plotClicked">
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QwtPlot" name="Qwt_Widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>50</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200000</width>
           <height>200000</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
